configuration/redis/redis.go

```diff
@@ -30,35 +28,38 @@ import (
 
 	"github.com/dapr/components-contrib/configuration"
 	"github.com/dapr/components-contrib/configuration/redis/internal"
+	rediscomponent "github.com/dapr/components-contrib/internal/component/redis"
 	contribMetadata "github.com/dapr/components-contrib/metadata"
 	"github.com/dapr/kit/logger"
 )
 
 const (
-	connectedSlavesReplicas   = "connected_slaves:"
-	infoReplicationDelimiter  = "\r\n"
-	host                      = "redisHost"
-	password                  = "redisPassword"
-	enableTLS                 = "enableTLS"
-	maxRetries                = "maxRetries"
-	maxRetryBackoff           = "maxRetryBackoff"
-	failover                  = "failover"
-	sentinelMasterName        = "sentinelMasterName"
-	redisDB                   = "redisDB"
-	defaultBase               = 10
-	defaultBitSize            = 0
-	defaultDB                 = 0
-	defaultMaxRetries         = 3
-	defaultMaxRetryBackoff    = time.Second * 2
-	defaultEnableTLS          = false
-	redisWrongTypeIdentifyStr = "WRONGTYPE"
+	connectedSlavesReplicas      = "connected_slaves:"
+	infoReplicationDelimiter     = "\r\n"
+	host                         = "redisHost"
+	password                     = "redisPassword"
+	enableTLS                    = "enableTLS"
+	redisMaxRetries              = "redisMaxRetries"
+	redisMaxRetryInterval        = "redisMaxRetryInterval"
+	redisMinRetryInterval        = "redisMinRetryInterval"
+	failover                     = "failover"
+	sentinelMasterName           = "sentinelMasterName"
+	redisDB                      = "redisDB"
```

Comment:
Does it make sense to keep those constants if are decoding from the `internal/metadata.Settings` struct? Besides, they are only used _once_ and on the test files, if you do need this constant, might be better to defined it there -- or honestly, not define it at all.

Reply:
Agreed - better to move to test only.